modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -1,83 +1,84 @@
-= Tracking Transmission Activity
+= Tracking Transmissions
 
-B2B transactions often communicate sensitive contractual obligations between companies. You can use Anypoint Partner Manager to monitor transmissions, view message contents, and store both transmission content and metadata.
+In Anypoint Partner Manager, one transmission can consist of one or more messages. Use the transmission view to monitor transmissions exchanged between partners and to view the transmission contents. At a high level, a transmission record contains the following information:
+
+* Data payloads received from partners or sent to partners via a configured protocol such as AS2, HTTP, SFTP or FTP.
+* Message disposition notifications (MDNs), if the payload is exchanged with partners via AS2 protocol
+functional acknowledgements.
+
+To track individual messages, see xref:activity-message-tracking.adoc[Tracking Individual Messages].
 
 == Locate a Transmission
 
 Select a transmission to view its details and view and download the message contents:
 
 . In the navigation menu, select *Activity*.
-. In the *Messages* field, select *Transmissions*.
+. Select *Transmissions*.
 . Search for the transmission.
-+
-When you search for a transmission, you can filter the results based on the following criteria:
+. Optionally filter the search results based on the following criteria:
 
-* Direction type
-* Date sent or received
+* Partner name
+* Message direction
+* Date range within which the message was sent
 +
-You can retrieve up to one week's worth of transmissions.
+You can retrieve up to one week's worth of messages.
 +
 * Acknowledgement status, such as `Overdue`
 * Message processing status, such as `Received`
-* Partner name
 * Partner message type, such as `X12 - 850`
 * Host message type, such as `JSON host-purchase-order`
-. Select the transmission to review summary information about the transmission and its message flow.
++
+Partner Manager returns a list of transmissions that meet your search criteria along with related information, such as the date the transmission was received, number of messages in the transmission, message type, control number, and acknowledgement status.
++
+* Click on a transmission to review additional information about the transmission and to view its message flow.
```

Comment:
I think this is another step, it should start with .
. Click on a transmission...